tools/create-test-data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
There's currently a `devel` directory with the ansible conf for Vagrant, maybe this script could be put in that directory instead of creating `tools`?